tests/integration/conftest.py

```diff
@@ -73,7 +79,10 @@ async def synapse_app_fixture(
         series="jammy",
         config={"server_name": server_name},
     )
-    await model.wait_for_idle(raise_on_blocked=True)
+    async with ops_test.fast_forward():
```

Comment:
Nice!

Reply:
Pardon my ignorance but, what's this for?

Reply:
This speeds up the `update-status` interval to 10s :D
```
Temporarily speed up update-status firing rate for the current model.
```